src/hooks/useItems.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import { ItemContext } from 'components/context'
+import { useFirestore, useFirestoreCollectionData } from 'reactfire'
+import { IItem } from 'interfaces'
+
+interface IUseItemsReturn {
+  collectionDocuments: { name: string; costPerUnit: number }[]
+  items: IItem[]
+  setItems: React.Dispatch<React.SetStateAction<IItem[]>>
+}
+
+export const useItems: (collectionName: string) => IUseItemsReturn = collectionName => {
+  const { items, setItems } = React.useContext(ItemContext)
+  const collectionRef = useFirestore().collection(collectionName)
+  const collectionDocuments = useFirestoreCollectionData<{ name: string; costPerUnit: number }>(collectionRef, {
```

Comment:
I thought we were turning this into a custom hook

Reply:
I thought I did?  I must be confused.  What do I need to do to make it a custom hook?  Maybe when can discuss this more when you have time.

Reply:
i'm an idiot